### DISCUSSION TITLE
Artboard previews not showing up at all, only upon hover

### DISCUSSION THREAD

**Discussion by sarudy89** [uxbox/uxbox#4790] [Q&A]
Hi, I have this issue, artboards are not previewing at all. Not even blurry. Instead I get some strange stripes. The issue is on both selfhosted and penpot.app official account. Both on Firefox and Brave. How can this be resolved? Thx

![pen_pot](https://github.com/penpot/penpot/assets/147134760/32b0b46a-e334-46f8-8e9c-a16880d51363)

**Comment by pascoual** [comment]
Same for me !
